core/src/test/java/io/kestra/core/runners/pebble/filters/StringFilter.java

```diff
@@ -0,0 +1,50 @@
+package io.kestra.core.runners.pebble.filters;
+
```

Comment:
Should this be a test file given that it is in the test folder?
The contents look to be duplicate as present in the above file.